<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>2340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMusic</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelBkg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>2340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/img/bkg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CProgressBar" name="barProgress" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2171</y>
      <width>990</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 255);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);</string>
    </property>
    <property name="value" stdset="0">
     <number>0</number>
    </property>
    <property name="textVisible" stdset="0">
     <bool>false</bool>
    </property>
    <property name="format" stdset="0">
     <string/>
    </property>
   </widget>
   <widget class="CButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>2201</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlay.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>2201</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPlayPrev">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>2211</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlayPrev.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPlayNext">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>2211</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlayNext.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelAlbumName">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>1680</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>2105</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="CLabel" name="labelSingerName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1570</y>
      <width>630</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="CButton" name="btnOrder">
    <property name="geometry">
     <rect>
      <x>965</x>
      <y>2215</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnOrder.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnRandom">
    <property name="geometry">
     <rect>
      <x>965</x>
      <y>2215</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnRandom.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelProgress">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2155</y>
      <width>990</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelPlayingfile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1500</y>
      <width>1000</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>2215</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnSetting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameDemand">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>840</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="CButton" name="btnDemandAlbum">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/medialib/demandalbum.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandAlbumItem">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/medialib/media.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandPlaylist">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/medialib/playlistset.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandSinger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/medialib/singerdefault.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandPlayItem">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/medialib/demandplayitem0.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frameDemandLanguage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>790</width>
       <height>60</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="CLabel" name="labelDemandCN">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>国语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandHK">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>130</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>粤语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandKR">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>130</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>韩语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandJP">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>130</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>日语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandEN">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>130</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>英文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandEUR">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>160</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>小语种</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>labelDemandHK</zorder>
     <zorder>labelDemandKR</zorder>
     <zorder>labelDemandJP</zorder>
     <zorder>labelDemandEN</zorder>
     <zorder>labelDemandEUR</zorder>
     <zorder>labelDemandCN</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X Music</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogoCompany">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>© Musicrossoft</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogoTip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="CButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnExit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnMore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnMore.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnFullScreen">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>25</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnFullScreen.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelSingerImg">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>1570</y>
      <width>633</width>
      <height>573</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>labelSingerImg</zorder>
   <zorder>labelBkg</zorder>
   <zorder>btnPlay</zorder>
   <zorder>btnPause</zorder>
   <zorder>btnPlayPrev</zorder>
   <zorder>btnPlayNext</zorder>
   <zorder>labelDuration</zorder>
   <zorder>btnOrder</zorder>
   <zorder>btnRandom</zorder>
   <zorder>btnSetting</zorder>
   <zorder>labelSingerName</zorder>
   <zorder>frameDemand</zorder>
   <zorder>labelLogo</zorder>
   <zorder>labelLogoCompany</zorder>
   <zorder>labelLogoTip</zorder>
   <zorder>barProgress</zorder>
   <zorder>btnMore</zorder>
   <zorder>btnExit</zorder>
   <zorder>btnFullScreen</zorder>
   <zorder>labelAlbumName</zorder>
   <zorder>labelPlayingfile</zorder>
   <zorder>labelProgress</zorder>
  </widget>
  <action name="action">
   <property name="text">
    <string>专辑</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">button.h</header>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">label.h</header>
  </customwidget>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">progressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
